sqlpage/templates/shell.handlebars

```diff
@@ -10,17 +10,56 @@
         <link rel="manifest" href="{{manifest}}">
     {{/if}}
     <link rel="stylesheet" href="{{static_path 'sqlpage.css'}}">
-   {{#each (to_array css)}}
+    {{#each (to_array css)}}
         {{#if this}}
             <link rel="stylesheet" href="{{this}}">
         {{/if}}
-   {{/each}}
+    {{/each}}
+
+    {{!-- Ensure font is an array --}}
+    {{#if font}}
+        {{#if (not (isArray font))}}
+            {{!-- Wrap single font string in an array --}}
+            {{#set "font" (array font)}}
+        {{/if}}
+    {{/if}}
+
+    {{#if font}}
+        <style>
+            {{#each font}}
```

Comment:
Can we split the support for multiple fonts and the support for local font files in two PRs ?

The support for local font files can be merged now. But as is, we are not ready for multi-font if we don't have a way to use them in components.

Reply:
hi, please check if it is ok now